CondCore/Utilities/scripts/conddb

```diff
@@ -2154,6 +2441,19 @@ def main():
     parser_listTags = parser_subparsers.add_parser('listTags', description='Lists all the Tags available in the DB.')
     parser_listTags.set_defaults(func=listTags_)
 
+    parser_listParentTags = parser_subparsers.add_parser('listParentTags', description='Lists all the Tags available in the DB, containing the matched payload hash.')
+    parser_listParentTags.add_argument('hash_name', help="Payload hash to match.")
+    parser_listParentTags.set_defaults(func=listParentTags_)
+
+    parser_diffGlobalTagsAtRun = parser_subparsers.add_parser('diffGloablTagsAtRun', description='Diffs two global tags, but showing only the differences relevant for a given run number.')
```

Comment:
The first argument of `add_parser` should be "diffGlobalTagsAtRun" rather than "diffGloablTagsAtRun".